app/models/manageiq/providers/azure/cloud_manager/security_group.rb

```diff
@@ -0,0 +1,2 @@
+class ManageIQ::Providers::Azure::CloudManager::SecurityGroup < ::SecurityGroup
+end
```

Comment:
Generally it's just for the STI typing ... it allows someone to ask for all of the Azure Security Groups without including other provider's security groups.